Tests/Get-BMVariable.Tests.ps1

```diff
@@ -260,14 +289,36 @@ Describe 'Get-BMVariable' {
         ThenNoErrorWritten
     }
 
+    It 'should return as value if variable is not OtterScript vector' {
+        GivenVariable 'Fubar' -WithValue 'Snafu'
+        WhenGettingVariable -ValueOnly
+        ThenVariableValuesReturned @( 'Snafu' )

```

Comment:
I'm not understanding this test. Why is the function returning an array if the variable is a scalar (string) value?